skyportal/models/gcn.py

```diff
@@ -187,6 +187,28 @@ class GcnEvent(Base):
         doc="Comments posted about this GCN event.",
     )
 
+    aliases = sa.Column(
+        sa.ARRAY(sa.String),
+        nullable=False,
+        server_default="{}",
```

Comment:
I'm not sure if this is the best way to define the default. Why not `default=[]`?

Reply:
Good point. I made that change. I also regenerated the migration script, and I'm wondering if I need to add server_defaults or not. It's not super clear to me what that does (does it set the value of new columns of all already existing rows to the server_default when migrating the DB?).

Reply:
I'm not sure either...